service/src/test/java/uk/nhs/adaptors/gp2gp/common/storageconnectors/StorageConnectorFactoryTest.java

```diff
@@ -0,0 +1,17 @@
+package uk.nhs.adaptors.gp2gp.common.storageconnectors;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@Slf4j
+public class StorageConnectorFactoryTest {
+
+    @Test
+    public void checkFactoryReturnsMockConnector() {
+        StorageConnector storageConnector = StorageConnectorFactory.getConfiguredConnector();
+        assertTrue(storageConnector instanceof LocalMockConnector);
```

Comment:
We could parametrise this test to assert other connectors.

Reply:
Once StorageConnectorFactory is setup as a spring factory bean we should also be over to override the configuration values - it would be easiest to have a separate StorageConnectorConfiguration class just to hold configuration values for this purpose.